public_html/layouts/basic/modules/Vtiger/resources/DetailPreview.js

```diff
@@ -12,15 +15,15 @@ Vtiger_Detail_Js("Vtiger_DetailPreview_Js", {}, {
 		});
 	},
 	/**
-	 * Function redirects to current iframe's parent
-	 * 
+	 * Redirects to the current iframe's parent.
 	 */
 	updateParentFrame: function () {
 		parent.app.getPageController().updateWindowHeight($(".mainContainer").height(), $(window.frameElement));
 	},
 	/**
-	 * Function sets correct size of iframes
-	 * 
+	 * Function sets the correct iframe's size.
+	 * @param {$} currentHeight - ifrmae's body height to be set.
```

Comment:
{$}

Reply:
I decided to use $ for jquery objects, what do you think about that?